RadarSDK/RadarEvent.m

```diff
@@ -299,6 +303,12 @@ - (instancetype _Nullable)initWithObject:(id)object {
         if (locationAccuracyObj && [locationAccuracyObj isKindOfClass:[NSNumber class]]) {
             NSNumber *locationAccuracyNumber = (NSNumber *)locationAccuracyObj;
 
+            // in indoors mode, override accuracy
+            RadarTrackingOptions *options = [Radar getTrackingOptions];
+            if(options.indoors){
+                locationAccuracyNumber = [NSNumber numberWithFloat:10.0];
```

Comment:
I think we want to do this server side. Best to not just throw away the data point